content/zh-cn/docs/contribute/style/diagram-guide.md

```diff
@@ -149,11 +149,11 @@ users, you will have many who are new to Kubernetes. Even a simple diagram can
 assist new users in absorbing Kubernetes concepts. They become emboldened and
 more confident to further explore Kubernetes and the documentation.  
 -->
-你需要考虑你的目标受众。除了一些有经验的 K8s 用户外，你还会遇到很多刚接触
+你需要考虑你的目标受众。除了一些有经验的 Kubernetes 用户外，你还会遇到很多刚接触
```

Comment:
Note: The English text is `K8s` instead of `Kubernetes`, not sure which is better here.

Reply:
In this small para, a reader can easily find 3 `Kubernetes` and 1 `k8s`, which is suggested to be consistent. If on different screens, it might be another case because few people may notice this minor inconsistency.